assets/javascripts/modules/ajaxFormSubmit.js

```diff
@@ -128,15 +135,21 @@ var ajaxFormSubmit = {
   },
 
   serializeForAjax: function(formScope) {
-    var ret = ['isajax=true'];
+    var result = ['isajax=true'];
```

Comment:
> Not really but in one of previous PRs it was mentioned to user more sensible name

sorry I meant all the work contained in `serializeForAjax` - if its not to do with this work then its best to split it out into a different PR